spec.bs

```diff
@@ -1450,7 +1457,7 @@ To <dfn>convert to an AuctionAd sequence</dfn> given a [=list=]-or-null |ads|:
 
 To <dfn>score and rank a bid</dfn> given an [=auction config=] |auctionConfig|, a [=generated bid=]
 |generatedBid|, a [=leading bid info=] |leadingBidInfo|, a [=string=] |decisionLogicScript|, a
-[=direct from seller signals=]-or-null |directFromSellerSignalsForSeller|, a {{unsigned long}}-or-null
+{{DirectFromSellerSignalsForSeller}} |directFromSellerSignalsForSeller|, a {{unsigned long}}-or-null

```

Comment:
nit: Unrelated to this PR, but `a {{unsigned long}}-or-null` should be `an {{unsigned long}}-or-null` (a -> an).

```suggestion
{{DirectFromSellerSignalsForSeller}} |directFromSellerSignalsForSeller|, an {{unsigned long}}-or-null

```